docs/source/installation.rst

```diff
@@ -0,0 +1,62 @@
+Installation
+============
+The qtconsole requires Qt, such as
```

Comment:
For consistency with other parts of the docs, let's say "The Qt console" rather than "The qtconsole".

As I see it:
* "The Qt console" is what we call the application
* "qtconsole" is the command that starts the application